python/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class LookNotFoundError(Exception):
```

Comment:
rename to `NotFoundError` .. and perhaps **Error** is redundant in the name, but that could be pythonic?

Reply:
I agree with you but unfortunately PEP8 suggests otherwise and it's what I'm following: https://www.python.org/dev/peps/pep-0008/#exception-names